src/index.css

```diff
@@ -11,3 +15,92 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+h3 {
+  width: auto;
+  display: flex;
+  font-weight: bolder;
+  background-color:  #c85c3b;
+  text-align: center;
+  color: #fff;
+  padding: 10px;
+  font-size: 14px;
+  margin: 25px 0 0 0; 
+  text-transform: uppercase;
+}
+
+
+h4 {
+  text-align: center;
+  padding: 7px;
+  border-radius: 5px;
+  font-size: 20px;
+  color: #ba4622;
+  font-weight: bold;
+}
+
+
+h5 {
+  font-weight: bold;
+  font-family: "Montserrat";
+  color: #ba4622;
+  line-height: 20px; 
+  font-size: 20px;
+  margin: 10px 0 0 10px; 
+  text-transform: uppercase;
+}
+
+p {
+ 
+  /* border: solid red 2px; */
+  color: black;
+  line-height: 30px; 
+  font-size: 16px;
+  /* display: flex; */
+  text-align: center;
+}
+    
+  @media (min-width: 668px) {
+h4{
+    font-size: 26px;
+    }
+
+  h3{
+      font-size: 16px;
+    }
+
+    p {
+      font-size: 20px;
+
+    }
+    
+  }
+    @media (min-width: 1024px) {
+
+      h4{
+        font-size: 28px;
+        }
+
+        h3{
+          font-size: 16px;
+        }
+    
+        p {
+          font-size: 20px;
+    
+        }
+        
+      }
+      
+
+b {
+  font-weight: bolder;
+  color: #ba4622;
+}
+
+
+a {
+
+  text-decoration: none;  
+}
```

Comment:
You can put this CSS in global instead, if you wan't. Totally fine to have it in CSS also, just different approaches,